client/src/api/axios-config.js

```diff
@@ -31,6 +32,25 @@ const addAuthHeader = async (config) => {
   return updatedConfig
 }
 
+const authReferesh = async (response) => {
+  const status = response ? response.status : null
+  if (status === 401) {
+    return refreshIDToken().then((newToken) => {
+      if (newToken) {
+        response.config.headers.Authorization = `Bearer ${newToken}`
+        response.config.baseURL = undefined
+        setToken(newToken)
+        return instance.request(response.config)
+      }
+      console.log('Token refresh non successful')
```

Comment:
before merging, you should remove the console.log to avoid clutter in the console